docs/en/latest/deployments/ack.md

```diff
@@ -38,6 +38,7 @@ This document explains how to install Ingress APISIX on [ali-cloud ACK](https://
 ```shell
 cd /path/to/apisix-helm-chart
 helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add apisix https://charts.apiseven.com
 helm dependency update ./charts/apisix
 helm install apisix ./charts/apisix \
```

Comment:
should not still use the local apisix, instead, use the online repo.

Reply:
> should not still use the local apisix, instead, use the online repo.

I have done the same as you said, isn't so?